examples/material_gallery/test/smoke_test.dart

```diff
@@ -62,12 +35,12 @@ Finder findBackButton(WidgetTester tester) => byTooltip(tester, 'Back');
 // Start a gallery demo and then go back. This function assumes that the
 // we're starting on home route and that the submenu that contains the demo
 // called 'name' is already open.
-void smokeDemo(WidgetTester tester, String menuItemText) {
+void smokeDemo(WidgetTester tester, String routeName) {
   // Ensure that we're (likely to be) on the home page
-  final Finder navigationMenuButton = findNavigationMenuButton(tester);
+  final Finder navigationMenuButton = byRouteName(tester, routeName);
```

Comment:
navigationMenu sounds like the drawer button or back button. we should rename this variable to, like, list item or something.